### PR TITLE
lxd/cluster: Handle lack of available target in evacuation

### PR DESCRIPTION
**Pull request by stgraber**

### CODE REVIEW

File: lxd/api_cluster.go [hunk 1]
```diff
@@ -2692,6 +2697,12 @@ func evacuateClusterMember(d *Daemon, r *http.Request) response.Response {
 				return err
 			}
 
+			// Skip migration if no target available.
+			if targetNodeName == "" {
+				logger.Info("No migration target available for instance", log.Ctx{"name": inst.Name(), "project": inst.Project()})
```
Comment (tomponline) [inline lxd/api_cluster.go:2702]: Should this be a warning?

Comment (stgraber) [reply lxd/api_cluster.go:2702]: Probably, I'll change it.

Comment (stgraber) [reply lxd/api_cluster.go]: Done!

**Comment by stgraber** [comment]
@tomponline should be good to go

**Pull request merged**